docs/system/k8s-metrics.md

```diff
@@ -85,6 +85,74 @@ This metric is [recommended][MetricRecommended].
 
 
 
+<!-- markdownlint-restore -->
+<!-- prettier-ignore-end -->
+<!-- END AUTOGENERATED TEXT -->
+<!-- endsemconv -->
+
+### Metric: `k8s.pod.network.io`
+
+This metric is [recommended][MetricRecommended].
+
+<!-- semconv metric.k8s.pod.network.io -->
+<!-- NOTE: THIS TEXT IS AUTOGENERATED. DO NOT EDIT BY HAND. -->
+<!-- see templates/registry/markdown/snippet.md.j2 -->
+<!-- prettier-ignore-start -->
+<!-- markdownlint-capture -->
+<!-- markdownlint-disable -->
+
+| Name     | Instrument Type | Unit (UCUM) | Description    | Stability |
+| -------- | --------------- | ----------- | -------------- | --------- |
+| `k8s.pod.network.io` | Counter | `By` | Pod network IO | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
+
+| Attribute  | Type | Description  | Examples  | [Requirement Level](https://opentelemetry.io/docs/specs/semconv/general/attribute-requirement-level/) | Stability |
+|---|---|---|---|---|---|
+| [`network.io.direction`](/docs/attributes-registry/network.md) | string | The network IO operation direction. | `transmit` | `Recommended` | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
+| [`system.device`](/docs/attributes-registry/system.md) | string | The device identifier | `(identifier)` | `Recommended` | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
```

Comment:
To save ourselves work later and to start preparing end users, can we start a list somewhere, maybe in the readme, of the breaking changes that will be included in the stable versions of these semantics?

I know this particular change is additive in this repo, but considering that the kubeletstatsreceiver has been emitting this metric with the attributes `interface` and `direction` since its inception this change will be very impactful.

Reply:
Good idea. I wonder where this information would fit best. @open-telemetry/specs-semconv-maintainers @open-telemetry/specs-semconv-approvers any suggestions?

Reply:
I think, ideally, we should be able to understand breaking changes and document them in Schema URL eventually for this purpose.

We don't have a good mechanism right now to do that, but @trask and @lmolkova have the most experience having stabilzied HTTP and soon, DB.

Reply:
One thing we can do for the Collector specifically is to file a meta issue that will collect all this breaking changes.

Additionally we could update the k8s' components that will be affected to have their documentation explicitly mentioning this breaking change meta issue.

Reply:
we usually have a migration plan as a part of stabilization process - like this one 

https://github.com/open-telemetry/semantic-conventions/blob/cf4ce09650a7a8bafc41a13d062e6f8ac64ade84/docs/database/README.md?plain=1#L18-L39

In a few words 
- the instrumentation should keep doing what it's doing now (following semconv or not)
- it should have an option to enable new conventions (for collector I'd assume it not be env var, but it'd make sense to have consistent way of opting in across different receivers). 
- it can drop old conventions or just switch to a new ones by default with the next major version. 

We've been also creating migration guides like these https://github.com/open-telemetry/semantic-conventions/blob/main/docs/non-normative/http-migration.md or https://github.com/open-telemetry/semantic-conventions/blob/main/docs/non-normative/db-migration.md which outline the list of changes. 

You can probably do something similar and just mention undocumented (in semconv) old attributes.

Reply:
Thank's for the information @lmolkova! If I understand correctly could we create a similar migration plan file for K8s already and start collecting the diffs there?

If I understood @TylerHelmuth initial comment here, we would like to start informing users already about such changes/diffs even if we are far from stability. In this way users could already be prepared about what changes to expect.

Reply:
@joaopgrassi I love it, thats exactly what I want to do here

Reply:
@ChrsMark where did we land with trying to track these breaking changes as we go?

Reply:
As mentioned at https://github.com/open-telemetry/semantic-conventions/pull/1427#issuecomment-2449304586, I have filed https://github.com/open-telemetry/semantic-conventions/issues/1537 for the migration plan (following the idea suggested above). I will start working on the migration plan PR soonish.

Reply:
Seems we are covered here. I'm going to resolve this conversation.